Alan_WarrenDesafio1/Program.cs

```diff
@@ -1,22 +1,29 @@
-using Alan_WarrenDesafio1.Data;
-using Alan_WarrenDesafio1.Validators;
-using AppServices;
+using Application.Validators;
+using Domain.Services;
 using FluentValidation.AspNetCore;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using System.Reflection;
 
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
-
 builder.Services.AddControllers()
     .AddFluentValidation(cfg =>
     {
-        cfg.RegisterValidatorsFromAssemblyContaining<CustomerValidator>();
+        cfg.RegisterValidatorsFromAssembly(Assembly.Load("Application"));
     });
+
 // Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
-builder.Services.AddSingleton<ICustomerServices, CustomerServices>();
+
+builder.Services.AddSingleton<ICustomerService, CustomerService>();
 builder.Services.AddTransient<ICustomerAppService, CustomerAppService>();
+var assemblie = new[] { Assembly.Load("Application") }; 
+builder.Services.AddAutoMapper((_, mapperConfiguration) => mapperConfiguration.AddMaps(assemblie), assemblie);
```

Comment:
Utilizar esse mesmo array de assemblies no register do FluentValidator

```suggestion
var assemblies = new[] { Assembly.Load("Application") }; 
builder.Services.AddAutoMapper((_, mapperConfiguration) => mapperConfiguration.AddMaps(assemblies), assemblies);
```